include/solvers/heat_transfer.h

```diff
@@ -190,6 +189,15 @@ class HeatTransfer : public AuxiliaryPhysics<dim, GlobalVectorType>
   void
   attach_solution_to_output(DataOut<dim> &data_out) override;
 
+  /**
+   * @brief Calculate delta_T_ref for the DCDD shock capture mechanism. delta_T_ref = T_max - T_min.
+   *
+   * @param minimum_delta_T_ref Limit the artificial viscosity added by the DCDD shock capture. DCDD shock capture elements are divided by delta_T_ref. An extremely low value of delta_T_ref will affect the results.
+   *
+   * @return The difference between the maximal temperature and the minimal temperature in the whole domain.
+   */
+  double
+  calculate_delta_T_ref(double minimum_delta_T_ref);
```

Comment:
If you want you can put a default value of 1, but otherwise, I think this is a good version of the function